pymine/types/buffer.py

```diff
@@ -339,8 +339,7 @@ def pack_ingredient(cls, ingredient: object) -> bytes:
         if isinstance(ingredient, (list, tuple)):
             if isinstance(ingredient[0], (dict, immutables.Map)):
                 return cls.pack_varint(len(ingredient)) + b"".join([cls.pack_slot(**slot) for slot in ingredient])
-            else:
-                return cls.pack_varint(len(ingredient)) + b"".join([cls.pack_slot(slot) for slot in ingredient])
+            return cls.pack_varint(len(ingredient)) + b"".join([cls.pack_slot(slot) for slot in ingredient])
```

Comment:
There should be a newline after the dedentation from the if statement

Reply:
i think this is resolved in main, so i removed it from mine?
i had some issues and had to delete and re-clone it